Cassette/CASFileObjectQueue.m

```diff
@@ -26,8 +26,12 @@ @interface CASFileObjectQueue ()
 @implementation CASFileObjectQueue
 
 - (instancetype)initWithRelativePath:(NSString *)filePath error:(NSError * __autoreleasing * _Nullable)error {
+    if ([filePath hasSuffix:@"/"]) {
```

Comment:
Is this the right way to check for a directory? User may have ended with a directory without using `/`

Reply:
the user may be creating a file/folder that doesn't exist, so in that case we can't really check if it's an existing folder. If it is an existing folder path, but without an ending `/`, since we append `.tmp` I believe to the stored file, it would still be a valid storage path.